rclrs/src/node/mod.rs

```diff
@@ -316,3 +315,20 @@ impl Node {
         domain_id
     }
 }
+
+// Helper used to implement run_string_getter(), but also used to get the FQN in the Node::new()
+// function, which is why it's not merged into Node::run_string_getter().
+// This function is unsafe since it's possible to pass in an rcl_node_t with dangling
+// pointers etc.
+unsafe fn run_string_getter_with_handle(
```

Comment:
Don't really know why, but the name seems odd. I'd name it `call_string_getter_with_handle`

Reply:
Done.